explorer/src/lib/dusk/string/decodeHexString.js

```diff
@@ -0,0 +1,31 @@
+import { hexToBytes, isValidHex } from "./";
+
+/**
+ * Decodes a hexadecimal string to utf-8 and formats it if it's a JSON string
+ *
+ * @param {string} value
+ * @returns {string}
+ */
+function decodeHexString(value) {
+  if (!isValidHex(value)) {
+    return value;
```

Comment:
Do we want to return the value/string or should we throw an error instead? 

Returning the value directly could be confusing to the caller since the function's purpose is to decode the hex string.

Reply:
Because the memo can be anything, I do not want the function to throw and error when called but still display the initial value.